model/simulationtests/outputcontrol_files.rb

```diff
@@ -44,23 +44,31 @@
 
 outputcontrol_files = model.getOutputControlFiles
 
+# If you set this to false, OpenStudio-workflow-gem issues a fatal since it is
+# checking the eplusout.end
+outputcontrol_files.setOutputEND(true)
+# If you set this to false, reporting measures will fail, including
+# openstudio_results
+outputcontrol_files.setOutputSQLite(true)
+# openstudio_results needs the tabular output since it queries it:
+# "Can't find any contents in Building Area Table to get tabular units. Measure can't run"
+outputcontrol_files.setOutputTabular(true)
```

Comment:
@joseph-robertson

Reply:
https://github.com/NREL/OpenStudio-resources/blob/e80f100244d44d8eb86bac58779b86f44ffba683/measures/openstudio_results/resources/os_lib_reporting.rb#L60-L67

Reply:
@jmarrec So should we remove `setOutputSQLite` and `setOutputEND` from hpp and have the ctor set to true for both?

Reply:
I dont think the sql one needs to be removed. If you don't use reporting measures then you don't need Sql. This object is advanced users only, use at your own risks.
The END one I'm leaning towards removing... it's an extremely small file and we'd need to change the workflow gem to not throw because of it. With loss of information back to user somewhat.

@kbenne @tijcolem Can we get your four cents, two each?

Reply:
I removed OutputEND from the idd, and getters/setters. Using OpenStudio-3.1.0-alpha+d1bd222025-Windows.exe, this test now passes.